docs/index.rst

```diff
@@ -6,11 +6,15 @@
 Welcome to ewatercycle's documentation!
 ==========================================================
 
+.. include:: ../README.rst
+
+User's Guide
+============
+
 .. toctree::
    :maxdepth: 2
-   :caption: Contents:
 
-.. include:: ../README.rst
+   observations
 
 Content
 =============
```

Comment:
This does not seem to be correct. `observations` under header `User's Guide` should be removed, see the [generated documentation of this PR](https://ewatercycle--164.org.readthedocs.build/en/164/). Also, the header `Content` can be replaced by the header `User's Guide`?

Reply:
Having 2 tocs make things weird. I dropped `User's guide` toc.